samples/cpp/benchmark_app/main.cpp

```diff
@@ -262,6 +262,11 @@ int main(int argc, char* argv[]) {
             config.erase(device);
         }
 
+        // If set batch size, disable the auto batching
+        if (FLAGS_b > 0) {
+            core.set_property(ov::hint::allow_auto_batching(false));
```

Comment:
Have you test it for devices which doesn't support this property?

Reply:
Yes, it also works in CPU（CPU doesn't support this property）